src/views/CompanyList.vue

```diff
@@ -139,12 +141,10 @@ export default {
   methods: {
     refetch() {
       this.$router.replace({ query: { ...this.$route.query, page: 1 } }).catch(() => {})
-      this.items = []
```

Comment:
I believe you do not need to reset the items array, because it will anyway be replaced after the refetch is completed, no?

Reply:
Looks good, agree no need to reset.
I think I had it because loader was at toverlay op and resetting cleared the lost and showed loader.
New overlay feels better!